src/main/java/org/onebusaway/alexa/AnonSpeechlet.java

```diff
@@ -270,31 +370,31 @@ private SpeechletResponse createOrUpdateUser(Session session,
         String outText = String.format("Ok, your stop number is %s in the %s region. " +
                         "Great.  I am ready to tell you about the next bus.  You can always ask me for arrival times " +
                         "by saying 'Alexa, open One Bus Away'.  Right now, %s",
-                stop.getStopCode(), region.getName(), arrivalInfoText);
+                stopCode, region.getName(), arrivalInfoText);
 
         Optional<ObaUserDataItem> optUserData = obaDao.getUserData(session);
         if (optUserData.isPresent()) {
             ObaUserDataItem userData = optUserData.get();
             userData.setCity(cityName);
-            userData.setStopId(stop.getId());
+            userData.setStopId(stopId);
             userData.setRegionId(region.getId());
             userData.setRegionName(region.getName());
             userData.setObaBaseUrl(region.getObaBaseUrl());
             userData.setPreviousResponse(outText);
-            userData.setLastAccessTime(System.currentTimeMillis());
```

Comment:
I'm not seeing this?
<img width="980" alt="screen shot 2016-07-17 at 9 44 30 am" src="https://cloud.githubusercontent.com/assets/7005336/16901828/2e1b51f6-4c03-11e6-9d19-d6cc13b9b1b0.png">